geode-docs/tools_modules/geode_for_redis.html.md.erb

```diff
@@ -17,25 +17,22 @@ limitations under the License.
 
 <% set_title(product_name, "for Redis") %>
 
-<%=vars.product_name%> for Redis allows <%=vars.product_name%> to function as a drop-in replacement for a
+<%=vars.product_name%> for Redis allows <%=vars.product_name%> to function as a
 highly-available Redis data store, letting Redis applications take advantage of
-<%=vars.product_name%>’s scaling capabilities without changing their client code. Redis clients connect to a <%=vars.product_name%>
-server in the same way they connect to a Redis server, using a hostname and a port number, with
-optional password authentication.
+<%=vars.product_name%>’s scaling capabilities with minimal changes to their client code.
 
-<img src="../images_svg/geode_for_redis.svg" class="image" />
 
 ## <a id="using-geode-for-redis"></a>Using <%=vars.product_name%> for Redis
+To use the <%=vars.product_name%> for Redis feature, the cluster must have at least one server that has enabled the feature to handle incoming Redis commands.
 
-The <%=vars.product_name%> cluster must have at least one server that is set up to handle the incoming Redis commands.
-
-Prerequisites for running the examples:
+Prerequisites:
 
 1. **Install <%=vars.product_name%>** <br/>
 Using the instructions in the `README.md` file in the root of the <%=vars.product_name%> checkout directory, build and install <%=vars.product_name%>.
 2. **Install the Redis CLI** <br/>
 Follow installation instructions at https://redis.io/download
```

Comment:
This wording might make it sound like a prerequisite for using geode-for-redis is to have the Redis CLI installed, which is not strictly true, it's just used in this example to ping the server. Maybe this could be changed to something along the lines of "have a Redis client so you can connect to the server" for the prerequisite, and then say "in this example we'll be using the Redis CLI, which can be installed from here..."?

Reply:
Agreed.  Changed and updated to reflect feedback.